wagtail/admin/ui/tables.py

```diff
@@ -264,3 +264,7 @@ def __repr__(self):
         @cached_property
         def classname(self):
             return self.table.get_row_classname(self.instance)
+
+
+class InlineActionsTable(Table):
+    classname = "listing listing--inline-actions"
```

Comment:
I also considered adding `classname` to one of `Table`'s `__init__()` parameters. Do you think we should do that instead?

Reply:
I think it would be useful to support ``classname`` as an argument to ``Table.__init__``, but in this case, I think sub-classing makes sense